### DISCUSSION TITLE
jquery-ui license type

### DISCUSSION THREAD

**Discussion by ligantx** [jquery/jquery-ui#2118] [Q&A] [answered]
I want to use jquery-ui in a commercial project. Am i allowed to do that for free?
Although jquery is licensed under MIT which is clear [here](http://jquery.org/license/), which means, that it can be used for free in commercial projects, i cannot find the license type that jquery-ui uses.

**Comment by TheWitness** [comment]
Pretty sure it's the same.

**Comment by markvantilburg** [comment]
https://github.com/jquery/jquery-ui/blob/main/LICENSE.txt

**Comment by ligantx** [comment]
> https://github.com/jquery/jquery-ui/blob/main/LICENSE.txt

so, although it's not defined as MIT, i guess its totally free to use for my commerial project:

**Permission** is hereby granted, **free of charge**, to any person obtaining
a copy of this software and associated documentation files (the
"Software"), to deal in the Software without restriction, including
without limitation the **rights to use**, copy, modify, merge, publish,
distribute, sublicense, and/or **sell copies** of the Software, and to
permit persons to whom the Software is furnished to do so, subject to
the following conditions:

**Comment by mgol** [comment, marked as answer]
This source has the MIT license as also declared in `package.json`:
https://github.com/jquery/jquery-ui/blob/5fa0db48d73887facf891eb05b43c443b5baeca3/package.json#L46

There are just exceptions for demos and external dependencies as spelled in the LICENSE.txt file.
